submissions/expero1/html-css-popup/style.css

```diff
@@ -0,0 +1,405 @@
+* {
+	margin: 0;
+	box-sizing: border-box;
+	font: 1em 'Arial', sans-serif;
+}
+*:focus,
+*:checked {
+	outline: 2px solid grey;
+}
+
+
+a {
+	text-decoration: none;
+	
+}
+a:hover, a:focus {
+	text-decoration: underline;
+}
+
+/* Header */
+header {
+	height: 50px;
+	justify-content: center;
+	background-color: whitesmoke;
+	box-shadow: 0 0 5px grey;
+
+	background-color: white;
+}
+
+.header-wrapper {
+position:relative;
+	max-width: 700px;
+	margin: 0 auto;
+	display: grid;
+	grid-template-columns: 1fr 8fr 50px 50px 50px;
+	grid-template-areas: "logo main-menu popup-menu-wrapper notification-menu user-menu ";
+	align-items: center;
+	gap: 20px;
+	vertical-align: center;
+	height: 100%;
+}
+.logo {
+	grid-area: logo;
+}
+
+/* Main menu */	
+.main-menu {
+	grid-area: main-menu;
+}
+.main-menu ul {
+	display: flex;
+	justify-content: flex-end;
+	align-items: center;
+	gap: 40px;
+margin-right:20px;
+}
+
+.main-menu li {
+	list-style: none;
+}
+
+.main-menu a {
+	color: gray;
+	text-transform: uppercase;
+	font-weight: bold;
+	letter-spacing: 1px;
+}
+
+/* Popup Button and */
+.menu-button {
+	
+	width: 30px;
+	height: 30px;
+	cursor: pointer;
+	border: none;
+}
+.popup-button {
+	display: block;
+	width: 30px;
+	height: 30px;
+	cursor: pointer;
+	border: none;
+	background: no-repeat center/90% url(./img/popup-button.png) ;
+}
+
+/*popup checkbox*/
+
+.popup-checkbox {
+	position: absolute;
+	-webkit-appearance: none;
+	-moz-appearance: none;
+	appearance: none;
+	outline: none;
+}
+
+
+.popup-checkbox:checked + label .popup-button,
+.popup-checkbox:focus + label .popup-button {
+	outline: 2px solid grey;
+	/*background: no-repeat center/100% url(./img/apps.png);*/
```

Comment:
Avoid commented code in your PRs

Reply:
Thank You. I have removed commented code.